app/uk/gov/hmrc/gform/eval/AllFormComponentExpressions.scala

```diff
@@ -27,8 +27,8 @@ import uk.gov.hmrc.gform.sharedmodel.formtemplate._
 object AllFormComponentExpressions extends ExprExtractorHelpers {
   def unapply(fc: FormComponent): Option[List[ExprMetadata]] = {
 
-    def toFirstOperandExprs(exprs: List[Expr], toExprMetadata: InferrableExpr => ExprMetadata): List[ExprMetadata] =
-      exprs.map(expr => toExprMetadata(InferrableExpr(expr, InferringRule.FirstOperand)))
+    def toFirstOperandExprs(exprs: List[Expr], toExprMetadata: Expr => ExprMetadata): List[ExprMetadata] =
```

Comment:
This method seems moot as it doesnt do anything specific i.e InferringRule.FirstOperand rule is no more. Caller can just replace this call with `exprs.map(toExprMetadata)`?